cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	fmt.Println(cmd.Path)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		log.Fatal(err)
+	}
+
+	in := bufio.NewScanner(stdout)
+
+	for i := 0; i < 20; i++ {
+		if in.Scan() {
+			log.Println(in.Text())
```

Comment:
Instead of printing `in.Text()` here, maybe check that the output is equal to what you expect, in this case "y"; similarly, I think you'll want to add another test that calls the yes command with some arguments, and then you'd want to make sure that `in.Text()` matches the arguments you put in
Looking at testutil.Command, you should just be able to add the arguments by doing `testutil.Command(t, "arg1", "arg2")`

Reply:
t.Errorf("scan: got failure, want no error")
break

Reply:
Fixed this! I added another function, which tested the yes command if I passed in multiple parameters. In this case, I passed in "hi how are you", and tested it across 1000 iterations to see if each iteration, I received the output "hi how are you".